compiler/rustc_codegen_llvm/src/gotoc/mir_to_goto/context/goto_ctx.rs

```diff
@@ -268,6 +279,21 @@ impl MetadataLoader for GotocMetadataLoader {
     }
 }
 
+/// The full crate name should use the Codegen Unit builder to include full name resolution,
```

Comment:
Is this where this function belongs?

Reply:
I could see an argument for either, where it's used for global names, or `compiler/rustc_codegen_llvm/src/gotoc/mir_to_goto/utils/names.rs`. Maybe the right thing is to move both the global name stuff and this to `names.rs`, thoughts?

Reply:
Yeah, I think that makes sense